docs/user-guide/test-zen-doc_Sean.md

```diff
@@ -53,50 +52,57 @@ FTP Port  |The FTP Port number for internal use. The default port is 21. If not
+Upload Zowe PAX for offline install | Upload a local (already downloaded) Zowe .PAX file using this option. <br /> &nbsp;&nbsp; 1. Click **UPLOAD PAX**.  <br /> &nbsp;&nbsp; 2. Select the downloaded PAX file and click **Open**.
+SMP/E | Use this option to install Zowe through SMP/E build outside the wizard. <br /> &nbsp;&nbsp; 1. Provide the location of the Runtime Directory. <br /> &nbsp;&nbsp; 2. Click **VALIDATE LOCATION**.                           <br /> &nbsp;&nbsp; **Note:** In the case of SMP/E installation, in the Initialization window, under the **Installation** tab, confirm the dataset names used during installation. <br /> &nbsp;&nbsp; 3. Click **Save**. 
 
 
-12. Click **Continue to Component Installation**.
+2. Click **Continue to Component Installation**.
 
 ## Zowe Server Configuration
 
-Each page of the wizard from this point onward will prompt you for configuration values for a particular step of Zowe server configuration.
-Some pages may require an additional administrator with sufficient privileges to complete the step. If you encounter such a page, you can **Skip** the page to allow the step to be completed externally to the wizard. It is also possible to input the correct values on the page, and use the **View Job** button to collect the job content to provide to an administrator without submitting it.
-Each value entered in a page will be validated, and when you complete entry on the page, the following actions can be performed:
+Perform Zowe server configuration in the wizard by providing inputs to the prompts for configuration values.
+Some steps may require an administrator with sufficient privileges to complete the step. You have the option to **Skip** the page to allow the step to be completed externally to the wizard. It is also possible to input the correct values on the page, and click **View Job** to collect the job content to provide to an administrator without submitting it.
+Each value entered is validated.
+
+When you complete an entry the following actions can be performed:
 
-* View/Edit YAML: This button lets you preview or adjust the YAML configuration that is used by Zowe. The prompts of the wizard are used to automatically generate its contents, but you can review it, edit it, or import and export its contents.
-* View/Submit Job: When you are happy with the values for the step, you can press the job submit button to execute the operations of the configuration step upon the z/OS system.
-* View Job output: Upon submission of the job, the output will be available for review.
+* **View/Edit YAML**  
+This feature lets you preview or adjust the YAML configuration that is used by Zowe. The prompts of the wizard are used to automatically generate the YAML contents, but you also have the options to review, edit, or import and export contents of the YAML file.
+* **View/Submit Job**  
```

Comment:
May need to change "View/Submit" to just "Submit"

Reply:
Yes @janan07 View Job feature is unavailable. all references should be gone for MVP. View Job Output is a feature, missing from Doc, and it is used to "view the submitted job output of an action in the wizard".

Reply:
"After a job is submitted, job output is available for review." also a good place to put this ^